src/frontend/store/actions/initialize-branding.ts

```diff
@@ -1,13 +1,37 @@
-import { BrandingOptions } from '../../../adminjs-options.interface'
+import { AdminJSOptions, BrandingOptions } from '../../../adminjs-options.interface';
```

Comment:
doesn't core repo's eslint enforce "no semicolon"?

Reply:
Good point, introduced prettier `"semi": false` and will be added in eslint in dev environment